apps/explorer/src/ui/header/NavItem.tsx

```diff
@@ -14,7 +14,7 @@ export const NavItem = forwardRef<HTMLButtonElement, NavItemProps>(
 		<button
 			ref={ref}
 			type="button"
-			className="flex cursor-pointer items-center gap-1 rounded-md border-none bg-transparent px-3.5 py-2 text-heading6 font-medium text-hero-darkest outline-none hover:bg-white/60 ui-open:bg-white/60"
+			className="flex h-10 cursor-pointer items-center gap-2 rounded-md border-none bg-transparent px-3.5 py-2 text-heading6 font-medium text-hero-darkest outline-none hover:bg-white/60 ui-open:bg-white/60"
```

Comment:
do we need explicit height here?

Reply:
I am not confident we can simply do this via paddings since the search bar and the network selection button have different dimensions. To make sure they are both 40px in height would be a lot messier than locking them at a fixed height. Open for suggestion though.